pages/etherpad/[[...roomId]].js

```diff
@@ -23,6 +23,34 @@ import CreateAuthoredPad from './actions/CreateAuthoredPad';
 import CreatePasswordPad from './actions/CreatePasswordPad';
 import { path as etherpadPath } from '../../lib/Etherpad';
 
+const WritePads = ({ serverPads, name, writeRoomId, etherpadId, ref, selected }) => {
```

Comment:
`WritePads` is a bit misleading, when this is only redering exactly **one** link for **one** pad.